jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/comments/CaseCommentsViewImpl.java

```diff
@@ -55,8 +58,17 @@
     Div commentsContainer;
 
     @Inject
+    @Bound
     @DataField("comments-list")
-    Div comments;
+    private ListComponent<CaseCommentSummary, CaseCommentItemView> comments;
+
+    @Inject
+    @AutoBound
+    private DataBinder<List<CaseCommentSummary>> caseCommentList;
+
+    @Inject
+    @DataField("empty-list-item")
+    private Div emptyContainer;
 
```

Comment:
There are two fields injected in this class which are not used anywhere. I think they should be deleted not to clutter the class unnecessarity: commentCreation and newCommentLabel